js/utility/mDDuration.js

```diff
@@ -0,0 +1,7 @@
+export default function calculateDuration(startDate) {
+  const endDate = new Date().getTime();
```

Comment:
prefer `Date.now()` over the `Date` constructor.

// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now

```suggestion
  const now = Date.now();
```

Reply:
God bless Internet Explorer 8, is dead. 😅